R/multitables.R

```diff
@@ -99,14 +155,20 @@ newMultitable <- function (formula, data, name, ...) {
         name <- formulaRHS(formula)
     }
 
+    u <- .newMultitable(template, shojiURL(data, "catalogs", "multitables"), name, ...)
+    
+    invisible(Multitable(crGET(u)))
+}
+
+## Internal function to do the POSTing, both in [[ and in newMultitable
+.newMultitable <- function (template, multiable_url, name, ...) {
```

Comment:
call the second arg `catalog_url`. I was confused when I first read it thinking that you were passing in a multitable entity url.